docs/proposals/resource_management.md

```diff
@@ -0,0 +1,313 @@
+/G/PodUID/cpu.quota = cpu request of Pod
+/G/m.limit_in_bytes = summation of memory limits of all guaranteed containers
+/G/PodUID/m.limit = memory limit of Pod <br>
+
+Example:
+We have two pods Pod1 and Pod2 having Pod Spec given below
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod1
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 10m
+                    memory: 1Gi
+        name: bar
+            resources:
+                limits:
+                    cpu: 100m
+                    memory: 2Gi
+```
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod2
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 20m
+                    memory: 2Gii
+```
+
+Pod1 and Pod2 are both classified as Guaranteed and are nested under the `/` cgroup.<br>
+
+/G/cpu.shares = 130m <br>
+/G/Pod1/cpu.quota = 110m <br>
+/G/Pod2/cpu.quota = 20m <br>
+/G/m.limit_in_bytes = 5Gi <br>
+/G/Pod1/m.limit = 3Gi <br>
+/G/Pod2/m.limit = 2Gi
+
+#### Burstable Cgroup ####
+
+We have the following resource parameters for the `Bu` cgroup.
+
+'/Bu/cpu.shares = summation of all cpu requests <br>
+/Bu/PodUID/cpu.quota = Pod Cpu Limit <br>
+/Bu/m.limit_in_bytes = Allocatable - summation of memory requests/limits of guaranteed pods <br>
+/Bu/PodUID/m.limit = Pod memory limit<br>
+
+`Note: For the Burstable Qos when limits are not specified for any one of the containers, the Pod limit defaults to the node resource capacity.`
+
+Example:
+We have two pods Pod3 and Pod4 having Pod Spec given below:
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod3
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 50m
+                    memory: 2Gi
+                requests:
+                    cpu: 20m
+                    memory: 1Gi
+        name: bar
+            resources:
+                limits:
+                    cpu: 100m
+                        memory: 1Gi
+```
+
+```yaml
+kind: Pod
+metadata:
+    name: Pod4
+spec:
+    containers:
+        name: foo
+            resources:
+                limits:
+                    cpu: 20m
+                    memory: 2Gi
+                requests:
+                    cpu: 10m
+                    memory: 1Gi  
+```
+
+Pod3 and Pod4 are both classified as Burstable and are hence nested under the Bu cgroup <br>
+/Bu/cpu.shares = 130m <br>
+/Bu/Pod3/cpu.quota = 150m <br>
+/Bu/Pod4/cpu.quota = 20m <br>
+/Bu/m.limit_in_bytes = Allocatable - 5Gi <br>
+/Bu/Pod3/m.limit = 3Gi <br>
+/Bu/Pod4/m.limit = 2Gi <br>
+
+#### Best Effort Cgroup ####
+
+For pods belonging to the Best effort Qos we don't specify any cpu parameters for the pod cgroup.<br>
+
+/BE/cpu.shares = 2 <br>
+/BE/cpu.quota= not set
+/BE/m.limit_in_bytes = Allocatable - summation of memory requests/limits of guaranteed pods <br>
+/BE/PodUID/m.limit = no limit <br>
```

Comment:
@vishh - nm, it makes sense as you described.  so lack of cpu would end up with shares=2.  In that case, if you just had a Bu and BE pod on the node, they would get equal CPU time.  I think that would be good to doc here or elsewhere as it takes some thought to recognize that lack of CPU will get you default min shares in the Bu class.